docs/DeveloperGuide.adoc

```diff
@@ -729,12 +729,82 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |use shorter aliases for commands|enter commands quickly and not have to type in the full command
+
+|`* * *` |user |find people by their tags|view people under certain tags
```

Comment:
`view people under certain tags` doesn't sound like a true benefit. It is a possible implementation.